index.html

```diff
@@ -1418,7 +1418,5 @@ <h2>Our Advice for Newbies!</h2>
       <a href="advice/Gabriela-L-advice.html.html">Gabriela Lopez's Advice</a><br>
       <a href="advice/kristen-t-adams-advice.html">Kristen T. Adams’ Advice</a><br>
       <a href="advice/katherine-siegel-advice.html">Katherine Siegel's Advice</a><br>
-      <a href="advice/sophie-j-rose-advice.html">Sophie's Advice</a><br>.
-    </section>
-  </body> 
-</html>
+      <a href="advice/sophie-j-rose-advice.html">Sophie's Advice</a><br>
+      <a href="advice/kelly-campbell-advice.html">⭐Kelly Campbell's Advice⭐</a><br>
```

Comment:
Hi @Kelly-Campbell,

Congrats on making your first pull request! 

It looks like the last three closing HTML tags have been removed, </section>, </body>, and </html>. Once you've added them back and completed the steps in the Finishing the Forking Workflow lesson, you can push your advice branch to GitHub and it will update your pull request automatically.

Reach out if you have any questions! 

Best,
Lisa

Reply:
Hi @twosavoie!  So I ended up completing the steps in the Finishing the Forking Workflow lesson before I saw your pull request message saying to add back in the last 3 closing HTML tags.  I went back & added them & then re-pushed to GitHub but not sure that I did that correctly.  I feel like my command line doesn't look correct.  
![image](https://github.com/skillcrush/git-practice/assets/172945977/14078374-b76e-4ff2-a736-76bb2b04f7df)

Reply:
Hi @twosavoie!  I went thru this again & added the last 3 closing html tags.  How can I tell if the pull request has been updated?  Also, this is now what my command line looks like.  What should I do from here?  Thank you!
![image](https://github.com/user-attachments/assets/1994b663-a8ea-4ff4-9751-aa50871ce4eb)